stdlib/3/tkinter/tix.pyi

```diff
@@ -0,0 +1,296 @@
+import tkinter
+from typing import Any, Dict, List, Optional, Tuple
+from typing_extensions import Literal
+
+WINDOW: Literal["window"]
+TEXT: Literal["text"]
+STATUS: Literal["status"]
+IMMEDIATE: Literal["immediate"]
+IMAGE: Literal["image"]
+IMAGETEXT: Literal["imagetext"]
+BALLOON: Literal["balloon"]
+AUTO: Literal["auto"]
+ACROSSTOP: Literal["acrosstop"]
+
+ASCII: Literal["ascii"]
+CELL: Literal["cell"]
+COLUMN: Literal["column"]
+DECREASING: Literal["decreasing"]
+INCREASING: Literal["increasing"]
+INTEGER: Literal["integer"]
+MAIN: Literal["main"]
+MAX: Literal["max"]
+REAL: Literal["real"]
+ROW: Literal["row"]
+S_REGION: Literal["s-region"]
+X_REGION: Literal["x-region"]
+Y_REGION: Literal["y-region"]
+
+TCL_DONT_WAIT: Literal[2]
+TCL_WINDOW_EVENTS: Literal[4]
+TCL_FILE_EVENTS: Literal[8]
+TCL_TIMER_EVENTS: Literal[16]
+TCL_IDLE_EVENTS: Literal[32]
+TCL_ALL_EVENTS: Literal[0]
```

Comment:
These constants are also in `_tkinter` stub, but with different names. Can you change the `_tkinter` constants to have `Literal` types too or change these constants to be `int`? Also, please a comment about keeping in sync to both places.